.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": ["airbnb-base", "prettier"],
```

Comment:
Depending on the supported Node version, `eslint-config-airbnb-base` might suggest syntax that is not yet available. Maybe you should add a legacy config or es5-targeted config here instead and remove the overrides below?

Reply:
The config you declare here is compatible with Node v4, provided you put `'use strict';` at the top of every file.